Update README example for 'animationIn/animationOut' options

palerdot opened the pull request:
The current README example for `animationIn/animationOut` which has comma separated values does not work. Following space separated classes work 
```javascript
animationIn: ["animated fadeIn"],
animationOut: ["animated fadeOut"]
```
I took reference from [example source code](https://github.com/teodosii/react-notifications-component/blob/master/samples/js/components/examples/AnimationExample.tsx#L99)

Also, README has two/three variations of sample snippets like using `["animate__animated", "animate__fadeIn"]` which is not inline with some of the examples elsewhere in the doc. I guess fixing this would also help people trying out this library.

teodosii commented:
Hey, thank you very much for the pull request.

Although you're right on this one, both `["animated fadeIn"]` and `["animated", "fadeIn"]` do the same. The code is looking for an array, whether you supply the first value as multiple strings or not, it's the same result.

This was a mistake I didn't notice at the beginning, and since then I haven't touched examples file, not until recently I've migrated to typescript (but have not deployed yet!)

Also, keep in mind the `samples` project uses version 3 of `animate.css`. The examples there need to use existing classes for `v3`, while docs mention `BEM` usage just because at that moment when trying this library, you're probably using `v4`. This is by means no mistake, I just didn't bother upgrading, no benefit at all. I've upgraded other packages which had security issues.

teodosii commented:
@palerdot Maybe you wanna help and modify the other mentions in the README that point to v3? I'd much appreciate your help.

palerdot commented:
Let me try and see if I can raise a PR with updated README. But I cannot assure a timeline. If that is ok, I will look into this and see if I can update the README with correct classes that works with recent library version.

palerdot commented:
@teodosii I have updated the PR with following this.
 - Added info on `animate.css v4` breaking change to include `animate__` prefix.
 - Updated the examples with classes with `animate__` prefix since this is the default behaviour from `v4`. Added comments in the snippet that the classes belong to `animate.css v4`

Also expanded on using `animate.compat.css` which imports classes with no prefix.

Please check the PR and see if it is inline with the expected usage. If you would like me to make further change please post here (and possibly maybe assign me to this PR so that I don't lose track of this PR).

teodosii commented:
I'm sorry for seeing it so late, I just didn't notice the notification. Looks great, thanks for your work! I've merged it!
